### ISSUE TITLE
Drop app.php in Encryption, Files External and Files Sharing in NC21

### ISSUE THREAD

**Issue by HouraisanNEET** [nextcloud/server#26500]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are interested into the same feature.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

**Is your feature request related to a problem? Please describe.**
My log entries is full of ``app.php`` message at the level of debug.

**Describe the solution you'd like**
Drop ``app.php`` in Encryption, Files External and Files Sharing.

**Describe alternatives you've considered**
```sh
find /path/to/nextcloud/apps/ -name "app.php" -delete
```

**Additional context**
```
{"reqId":"lNWQw8TGaqBQCbnH6JPg","level":0,"time":"2021-04-11T12:12:36+00:00","remoteAddr":"127.0.0.1","user":"User","app":"files_sharing","method":"GET","url":"/nextcloud/apps/logreader/poll?lastReqId=CvRD55x7iXzZvtNS8DI1","message":"/appinfo/app.php is deprecated, use \\OCP\\AppFramework\\Bootstrap\\IBootstrap on the application class instead.","userAgent":"Mozilla/5.0 (X11; Linux x86_64; rv:78.0) Gecko/20100101 Firefox/78.0","version":"21.0.1.1"}
{"reqId":"KggYIaALdY8g2bHSxxX6","level":0,"time":"2021-04-11T12:12:36+00:00","remoteAddr":"127.0.0.1","user":"User","app":"files_sharing","method":"GET","url":"/nextcloud/ocs/v2.php/apps/serverinfo/api/v1//basicdata?format=json","message":"/appinfo/app.php is deprecated, use \\OCP\\AppFramework\\Bootstrap\\IBootstrap on the application class instead.","userAgent":"Mozilla/5.0 (X11; Linux x86_64; rv:78.0) Gecko/20100101 Firefox/78.0","version":"21.0.1.1"}
{"reqId":"JrBP7yUKyPLWYsTkQbfA","level":0,"time":"2021-04-11T12:12:37+00:00","remoteAddr":"127.0.0.1","user":"User","app":"files_sharing","method":"GET","url":"/nextcloud/apps/serverinfo/update","message":"/appinfo/app.php is deprecated, use \\OCP\\AppFramework\\Bootstrap\\IBootstrap on the application class instead.","userAgent":"Mozilla/5.0 (X11; Linux x86_64; rv:78.0) Gecko/20100101 Firefox/78.0","version":"21.0.1.1"}
{"reqId":"aKW0w1oLwQPg0Nm9fgK2","level":0,"time":"2021-04-11T12:12:37+00:00","remoteAddr":"127.0.0.1","user":"User","app":"files_sharing","method":"GET","url":"/nextcloud/apps/logreader/poll?lastReqId=lNWQw8TGaqBQCbnH6JPg","message":"/appinfo/app.php is deprecated, use \\OCP\\AppFramework\\Bootstrap\\IBootstrap on the application class instead.","userAgent":"Mozilla/5.0 (X11; Linux x86_64; rv:78.0) Gecko/20100101 Firefox/78.0","version":"21.0.1.1"}
{"reqId":"fWFiqistk5vblIzbV2go","level":0,"time":"2021-04-11T12:12:37+00:00","remoteAddr":"127.0.0.1","user":"User","app":"files_sharing","method":"GET","url":"/nextcloud/apps/logreader/poll?lastReqId=aKW0w1oLwQPg0Nm9fgK2","message":"/appinfo/app.php is deprecated, use \\OCP\\AppFramework\\Bootstrap\\IBootstrap on the application class instead.","userAgent":"Mozilla/5.0 (X11; Linux x86_64; rv:78.0) Gecko/20100101 Firefox/78.0","version":"21.0.1.1"}
```

**Comment by acsfer** [comment]
Duplicate of https://github.com/nextcloud/server/issues/23046

**Comment by joshtrichards** [comment]
Fixed in #46136
